### PR TITLE
Special Issue Report email

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

**Comment by diversemix** [comment]
Output emailed is something like:

image

**Comment by de-code** [comment]
FYI postgres has already CSV generation built in (which could save you some awk), e.g.:

```bash
psql -c "COPY (<select query>) TO STDOUT WITH CSV > special-issue.csv"
```

And then you could run a command like:

```bash
bq load \
    --project elife-data-pipeline
    --autodetect \
    --source_format=CSV \
    xpub_temp.special_issue \
    special-issue.csv
```

**Comment by diversemix** [comment]
> FYI postgres has already CSV generation built in (which could save you some awk), e.g.:
> 
> ```shell
> psql -c "COPY (<select query>) TO STDOUT WITH CSV > special-issue.csv"
> ```
> 
> And then you could run a command like:
> 
> ```shell
> bq load \
>     --project elife-data-pipeline
>     --autodetect \
>     --source_format=CSV \
>     xpub-temp.special_issue \
>     special-issue.csv
> ```

How could this save me some awk - the awk is there to convert the delimited file into an HTML table?

**Comment by de-code** [comment]
> How could this save me some awk - the awk is there to convert the delimited file into an HTML table?

Because then it would be in BigQuery and a Data Studio report could be created from it. What would the HTML table still be for?

**Comment by diversemix** [comment]
> > How could this save me some awk - the awk is there to convert the delimited file into an HTML table?
> 
> Because then it would be in BigQuery and a Data Studio report could be created from it. What would the HTML table still be for?

Still don't understand how this would save me the awk. The HTML table now sends weekly email to editors.

**Comment by diversemix** [comment]
@de-code ... Why not raise another issue to change it use bq ... at the moment this is not required/necessary  and is more work.

**Comment by de-code** [comment]
> > > How could this save me some awk - the awk is there to convert the delimited file into an HTML table?
> > 
> > 
> > Because then it would be in BigQuery and a Data Studio report could be created from it. What would the HTML table still be for?
> 
> Still don't understand how this would save me the awk. The HTML table now sends weekly email to editors.

A Data Studio report can now send scheduled email and is more flexible. My understanding is that this is purely a temporary solution because the data is currently not in BigQuery and there is no API to "properly" retrieve it.

File: scripts/special-issues.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This script is designed to be called from 'script-mailer.sh'
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+HOST=prod--xpub--1.elifesciences.org
+
+SEARCH_TERM=neuroblast
```
Comment (Cooryd) [inline scripts/special-issues.sh:9]: Would it be useful to make this an arg for future special issues?

**Review by Cooryd — APPROVED** [review]

File: scripts/special-issues.sh (hunk 1 above)
Comment (will-byrne) [reply scripts/special-issues.sh:9]: +1

**Review by will-byrne — APPROVED** [review]

**Review by erezmus — APPROVED** [review]

**Pull request merged**